src/cfp.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Call For Proposal
```

Comment:
It should be Call For Proposals (plural), that's what Pretalx seems to be using internally and also we have used that in past.